### PR TITLE
DashboardSettings: Fixes issue with tags list not updating changes are made 

### PR DESCRIPTION
**Pull request by huynhsamha**
**What this PR does / why we need it**:

I've created an issue about a bug from version 7.5.x, that when we add/remove tags in dashboard settings, the UI is not updated. 

In this PR, I've resolved this issue.

**Which issue(s) this PR fixes**: Issue #32188

Fixes #

+ Trigger rendering the new state on tags changes.

**Special notes for your reviewer**:

Demo for the bug:

**Before**

111903395-ad1b0280-8a74-11eb-9d5b-49f2832dd29b

**After**

Grafana - Demo fix tags v7 5 x

Please review my changes in this PR.

Many thanks to the Grafana team!

### CODE REVIEW

**Comment by zoltanbedi** [comment]
@jackw can you check this pr please?

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: docs/sources/administration/configuration.md [hunk 1]
```diff
@@ -268,6 +268,11 @@ Specify a full HTTP URL address to the root of your Grafana CDN assets. Grafana
 For example, given a cdn url like `https://cdn.myserver.com` grafana will try to load a javascript file from
 `http://cdn.myserver.com/grafana-oss/v7.4.0/public/build/app.<hash>.js`.
 
+### read_timeout
+
+Sets the maximum time using a duration format (5s/5m/5ms) before timing out read of an incoming request and closing idle connections.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:273]:
```suggestion
Sets the maximum time using the duration format (5s/5m/5ms) before reading of incoming request times out and idle connections close.
```

File: docs/sources/administration/configuration.md [hunk 2]
```diff
@@ -1082,6 +1091,12 @@ Configures max number of alert annotations that Grafana stores. Default value is
 
 <hr>
 
+## [annotations]
+
+### cleanupjob_batchsize
+
+Configures the batch size for the annotation clean-up job. This setting is used for dashboard, API, and alert annotations.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:1098]:
```suggestion
Configures the batch size for the annotation cleanup job. This setting is used for dashboard, API, and alert annotations.
```

File: docs/sources/datasources/cloudwatch.md [hunk 3]
```diff
@@ -375,6 +375,22 @@ To request a quota increase, visit the [AWS Service Quotas console](https://cons
 
 Please see the AWS documentation for [Service Quotas](https://docs.aws.amazon.com/servicequotas/latest/userguide/intro.html) and [CloudWatch limits](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html) for more information.
 
+## Configure the data source with grafana.ini
+
+In the [Grafana configuration](https://grafana.com/docs/grafana/latest/administration/configuration/#aws) there's an `AWS` section that allows you to customize the data source.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/cloudwatch.md:380]:
```suggestion
In the [Grafana configuration](https://grafana.com/docs/grafana/latest/administration/configuration/#aws) file, the `AWS` section allows you to customize the data source.
```

File: docs/sources/whatsnew/whats-new-in-v7-5.md [hunk 4]
```diff
@@ -102,3 +103,13 @@ If you enable the feature, then you can use template variables in reports.
...
 There are no known breaking changes in this release.
+
+## Updated configuration
+
+```
+[server]
+read_timeout = 0
+```
+
+Sets the maximum time using a duration format (5s/5m/5ms) before timing out read of an incoming request and closing idle connections.
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v7-5.md:114]:
```suggestion
Sets the maximum time using the duration format (5s/5m/5ms) before reading of incoming request times out and idle connections close.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]

**Comment by achatterjee-grafana** [comment]
Added some copy-edit suggestions.

**Comment by torkelo** [comment]
This needs a rebase ontop of master

**Comment by torkelo** [comment]
Bug fixes go into master, then get's backported to release branches

**Comment by torkelo** [comment]
Can you open a PR based on master?

**Comment by huynhsamha** [comment]
Hi @torkelo, I've created a new PR based on master #32241.

**Pull request closed without merging**